corehq/messaging/scheduling/tests/test_content.py

```diff
@@ -42,3 +44,67 @@ def test_custom_content(self):
                     call(schedule_instance, self.user, '9990000000000', 'Message 1'),
                     call(schedule_instance, self.user, '9990000000000', 'Message 2'),
                 ])
+
+    def test_get_translation_from_message_dict(self):
+        message_dict = {}
+        schedule = Schedule(domain=self.domain)
+
+        # Empty message dict results in no message
```

Comment:
This test looks like multiple tests, one for each place you have a comment like this. What do you think of starting a new test method at each comment?

Reply:
i had originally done it this way because the inputs to each test build off of each other, but agree splitting it up is better: https://github.com/dimagi/commcare-hq/pull/18840/commits/bd16e0a5fcd1c44e196a55f84643fddfb531b557